deploy/images/README.md

```diff
@@ -14,14 +14,15 @@ The result is an image, named `egarchive/lega-base`, and containing `python 3.6`
 
 ## Dependencies
 
-The following images are pulled from the main Docker hub:
+The following images are pulled from Docker Hub:
 
-* `egarchive/lega-inbox` (based on OpenSSH version 7.8p1 and CentOS7)
 * `egarchive/lega-mq` (based on `rabbitmq:3.6.14-management`)
 * `egarchive/lega-db` (based on `postgres:11.2`)
+* `egarchive/lega-inbox` (based on OpenSSH version 7.8p1 and CentOS7)
+* `nbisweden/lega-inbox` (based on Apache Mina)
 * `python:3.6-alpine3.8` 
 
-The [`egarchive/lega-inbox`](https://github.com/EGA-archive/LocalEGA-inbox) is a LocalEGA inbox, fetching user credentials from CentralEGA and sending file events notifications to the configured message broker. It is based on OpenSSH SFTP server version `7.8p1` 
+The [`egarchive/lega-inbox`](https://github.com/EGA-archive/LocalEGA-inbox) and [`nbisweden/lega-inbox`](https://github.com/NBISweden/LocalEGA-inbox) are LocalEGA inboxes, fetching user credentials from CentralEGA and sending file events notifications to the configured message broker. It is based on OpenSSH SFTP server version `7.8p1` 
```

Comment:
It is a bit confusing with the two `lega-inbox` with the same name but at different repos. I guess it will be better to rename one of them. But at the moment, probably just specify that the one from `egarchive` is the C implementation and the other one is the Java (?) implementation.

Reply:
One is called `egarchive/lega-inbox` and the other one is `nbisweden/lega-inbox`. I don't think it matters too much, since it is documentation for developers, not users. Those 2 names are fine enough, I think.

Reply:
ok